tests/system/python/api/test_statistics.py

```diff
@@ -150,7 +150,7 @@ def test_statistics_history_with_service_enabled(self, start_south_coap, fledge_
         assert 1 == stats['READINGS']
 
         # Allow stats collector schedule to run i.e. by default 15s
-        time.sleep(wait_time * 2 + 1)
+        time.sleep(wait_time * 3)
```

Comment:
In Postgres specific CI job, what is the wait-time arg value?

Reply:
> what is the wait-time arg value

CI used default i.e 5 seconds for python based tests except the `E2E`. And this issue is NOT only with Postgres we see with SQLite as well intermittently on Ubuntu Platform, Rpi bullseye but constantly fails on Rpi buster, CentOS Stream9